pkg/cmd/reset.go

```diff
@@ -103,27 +122,83 @@ func runReset(opts *resetOpts) error {
 		return err
 	}
 
-	if opts.DestroyWorkers {
+	if opts.DestroyWorkers || opts.CleanupVolumes || opts.CleanupLoadBalancers {
 		if cErr := kubeconfig.BuildKubernetesClientset(s); cErr != nil {
 			s.Logger.Errorln("Failed to build the Kubernetes clientset.")
-			s.Logger.Warnln("Unable to list and delete machine-controller managed nodes.")
-			s.Logger.Warnln("You can skip this phase by using '--destroy-workers=false' flag.")
-			s.Logger.Warnln("If there are worker nodes in the cluster, you might have to delete them manually.")
+			if opts.CleanupLoadBalancers {
+				s.Logger.Warnln("Unable to list and delete load balancers in the cluster.")
+				s.Logger.Warnln("You can skip this phase by using '--cleanup-load-balancer=false'.")
+				s.Logger.Warnln("If there are load balancers in the cluster, you might have to delete them manually.")
+			}
+			if opts.CleanupVolumes {
+				s.Logger.Warnln("Unable to list and delete dynamically provisioned and unretained volumes in the cluster.")
+				s.Logger.Warnln("You can skip this phase by using '--cleanup-volumes=false'.")
+				s.Logger.Warnln("If there are unretained volumes in the cluster, you might have to delete them manually.")
+			}
+			if opts.DestroyWorkers {
+				s.Logger.Warnln("Unable to list and delete machine-controller managed nodes.")
+				s.Logger.Warnln("You can skip this phase by using '--destroy-workers=false' flag.")
+				s.Logger.Warnln("If there are worker nodes in the cluster, you might have to delete them manually.")
+			}
+			s.Logger.Warnln("If there are unretained volumes and load balancers in the cluster, you might have to delete them manually.")
 
 			return cErr
 		}
 	}
 
-	s.Logger.Warnln("This command will PERMANENTLY destroy the Kubernetes cluster running on the following nodes:")
-
-	for _, node := range s.Cluster.ControlPlane.Hosts {
-		fmt.Printf("\t- reset control plane node %q (%s)\n", node.Hostname, node.PrivateAddress)
+	if opts.CleanupLoadBalancers {
```

Comment:
I would instead of listing each Load Balancer and volume just write that we'll delete all Load Balancer Services and PersistentVolumes/PersistentVolumeClaims.